lib/fluent/plugin/out_forward.rb

```diff
@@ -76,6 +81,19 @@ class ConnectionClosedError < Error; end
     desc 'Compress buffered data.'
     config_param :compress, :enum, list: [:text, :gzip], default: :text
 
+    desc 'The default version of TLS transport.'
+    config_param :tls_version, :enum, list: Fluent::PluginHelper::Socket::TLS_SUPPORTED_VERSIONS, default: Fluent::PluginHelper::Socket::TLS_DEFAULT_VERSION
```

Comment:
As you mentioned, these parameters are named in different way from servers' `<transport>` section.
I think that almost users just specify one or (at most) two parameters in these `tls_` prefixed parameters (`tls_cert_path`, and sometimes `tls_allow_self_signed_cert`). Many users don't change the default TLS version nor TLS ciphers.
In such cases, `tls_cert_path ...` is much more simple than `<tls> tls_cert_path ... </tls>`.